packages/frontend/app/routes/payments.outgoing.$outgoingPaymentId.tsx

```diff
@@ -0,0 +1,224 @@
+              <div>
+                {outgoingPayment.metadata ? (
+                  <>
+                    <button
+                      className='font-medium mb-1 cursor-pointer'
+                      aria-label='toggle metadata visibility'
+                      onClick={() => setShowMetadata(!showMetadata)}
+                    >
+                      {showMetadata ? '▼' : '►'} Metadata
+                    </button>
+                    {showMetadata && outgoingPayment.metadata && (
+                      <pre
+                        className='mt-1 text-sm'
+                        dangerouslySetInnerHTML={{
+                          __html: prettify(outgoingPayment.metadata)
+                        }}
+                      />
+                    )}
+                  </>
+                ) : (
+                  <>
+                    <p className='font-medium'>Metadata</p>
+                    <p className='mt-1'>
+                      <em>None</em>
+                    </p>
+                  </>
+                )}
+              </div>
+            </div>
+          </div>
+        </div>
+        {/* Outgoing Payment General Info - END */}
+
+        {/* Outgoing Payment Liquidity */}
+        <div className='grid grid-cols-1 py-3 gap-6 md:grid-cols-3 border-b border-pearl'>
+          <div className='col-span-1 pt-3'>
+            <h3 className='text-lg font-medium'>Liquidity Information</h3>
+          </div>
+          <div className='md:col-span-2 bg-white rounded-md shadow-md'>
+            <div className='w-full p-4 flex justify-between items-center'>
+              <div>
+                <p className='font-medium'>Amount</p>
+                <p className='mt-1'>
+                  {formatAmount(
+                    outgoingPayment.liquidity ?? '0',
+                    outgoingPayment.receiveAmount.assetScale
+                  )}{' '}
+                  {outgoingPayment.receiveAmount.assetCode}
+                </p>
+              </div>
+              <div className='flex space-x-4'>
+                {BigInt(outgoingPayment.liquidity ?? '0') ? (
+                  <Button
+                    aria-label='withdraw outgoing payment liquidity page'
+                    preventScrollReset
+                    to={`/payments/outgoing/${outgoingPayment.id}/withdraw-liquidity`}
+                  >
+                    Withdraw
+                  </Button>
+                ) : (
+                  <Button
+                    disabled={true}
+                    aria-label='withdraw outgoing payment liquidity page'
+                  >
+                    Withdraw
+                  </Button>
+                )}
+                {outgoingPayment.state === OutgoingPaymentState.Funding ? (
+                  <Button
+                    aria-label='deposit outgoing payment liquidity page'
+                    preventScrollReset
+                    to={`/payments/outgoing/${outgoingPayment.id}/deposit-liquidity`}
```

Comment:
Just using this line as an example, but it applies to many places.

"deposit" is consistent with the underlying mutation (`depositOutgoingPaymentId`) but elsewhere we're calling it "Add" liquidity (peers, assets which are themselves consistent with the underlying mutations). Max and I expressed a preference for just using deposit everywhere but I just wanted to get some more opinions before making such a change.
 https://github.com/interledger/rafiki/pull/2151#discussion_r1406673266 
 
There is also a case to be made for a different name in the UI (such as "Add") but I still think I like Deposit. I will make the displayed term consistent on the frontend in this PR one way or the other.

Reply:
Hmmm. Deposit makes sense for sure. Given the area we are working in, makes more sense then Add.